extractor/bug_info_test.go

```diff
@@ -31,7 +31,7 @@ func must(err error) {
 func TestValidateCommentBody(t *testing.T) {
 	owner := "pingcap"
 	repo := "tidb"
-	issueNum := 18792
+	issueNum := 19929
```

Comment:
I perfer directly put the body message in the test ,in order to make understanding the code simpler.

Reply:
That's just for convinence. There is a tset use raw string as input. I shall remove some redundant tests.